aws-emrserverless-application/aws-emrserverless-application.json

```diff
@@ -200,6 +200,10 @@
         "Disk": {
           "description": "Per worker Disk resource. GB is the only supported unit and specifying GB is optional",
           "$ref": "#/definitions/DiskSize"
+        },
+        "DiskType": {
+          "description": "Per worker DiskType resource. Shuffle optimized and Standard are only supported types and specifying diskType is optional",
```

Comment:
Do we want to say that it defaults to Standard when unspecified?

Reply:
Same here. auto generated docs based on model